app/controllers/api/v1/events_controller.rb

```diff
@@ -13,9 +13,9 @@ def show
       end
 
       def create
-        event = Event.new(event_params)
+        event = Event::Create.call(event_params.merge!(user_ids: [current_user.id]))
```

Comment:
`user_ids` should contain array of users that are connected to event. Plese do not override user_ids, just pass current user id as a separate class parameter and create `EventUser` record separatly in service.

Reply:
Changed parameters to 
`event = Event::Create.call(event_params, current_user)`